docker/entrypoint.sh

```diff
@@ -4,12 +4,17 @@ set -e
 
 if [ "$1" = 'celestia' ]; then
     echo "Initializing Celestia Node with command:"
-    echo "celestia "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}""
-    echo ""
 
-    celestia "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}"
+    if [[ -n "$NODE_STORE" ]]; then
+        echo "celestia "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}" --node.store "${NODE_STORE}""
+        celestia "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}" --node.store "${NODE_STORE}"
+    else
+        echo "celestia "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}""
+        celestia "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}"
+    fi
 
     echo ""
+    echo ""
```

Comment:
why?

Reply:
To separate the logs between `init` and `start` when starting the container.

```
...
NAME: my_celes_key
ADDRESS: celestia1gn5ed7fatxpn2ate7vfsd0wwz6vfvc8jk4hjah
MNEMONIC (save this somewhere safe!!!): 
*** *** *** *** *** *** ***


Starting Celestia Node with command:
celestia light start
...
```

I had issues when debugging something, so it's helpful to have some separation between them.